xml/System.Collections.Specialized/OrderedDictionary.xml

```diff
@@ -310,6 +310,7 @@
  ]]></format>
         </remarks>
         <exception cref="T:System.NotSupportedException">The <see cref="T:System.Collections.Specialized.OrderedDictionary" /> collection is read-only.</exception>
+        <exception cref="T:System.ArgumentException">The <see cref="T:System.Collections.Specialized.OrderedDictionary.Add%2A" /> tries to add a key that already exist.</exception>
```

Comment:
This should be exists rather than exist. But it would be better to not mention the Add method again and instead describe the exception condition as: "An element with the same key already exists in the \<see cref="T:System.Collections.Specialized.OrderedDictionary"> collection."